pylintrc

```diff
@@ -12,7 +12,7 @@ good-names=f,e,i,j,k,v,ex,_,x,y,z,kw
 # No camel case globals, max length 30
 const-rgx=(^([A-Za-z_](?!(([A-Z][a-z0-9]+)+))){2,30}$)
 max-args = 7
-max-attributes = 8
+max-attributes = 9
```

Comment:
Sorry that I just increased this, but I wanted to introduce the changes in the least intrusive way I could, and keeping the max-attribute at 8 would have required some refactoring in notifier.py not related to the change proposed in this pull request.